### ISSUE TITLE
[Android] Crash when navigating to task list

### ISSUE THREAD

**Issue by nickrandolph** [unoplatform/uno.todo#320]
![image](https://user-images.githubusercontent.com/1614057/169751969-27f526c9-582e-41a9-ae45-43a3eea60779.png)

cc @agneszitte-nventive @kazo0

**Comment by nickrandolph** [comment]
See https://github.com/unoplatform/uno.todo/pull/319 that has the latest uno + toolkit + material

**Comment by kazo0** [comment]
Yeah there seems to be an issue with using converters for the properties of AppBarButton and NavigationBar on iOS / Android. The native renderers apparently aren't handling this correctly but I still haven't found out why
